src/main/java/com/conveyal/r5/analyst/ResultSet.java

```diff
@@ -230,6 +234,53 @@ public void writeIsochrones (JsonGenerator jgen) throws IOException {
         jgen.writeEndArray();
     }
 
+    /**
+     * Reduces and simplifies (using DouglasPeuckerSimplifier) the inputted geometry.
+     *
+     * @param geometry  The geometry to modify
+     * @return          The new geometry that is simplified and then reduced
+     */
+    private Geometry reduceAndSimplifyGeometry(Geometry geometry) {
+        PrecisionModel precisionModel = new PrecisionModel(10000);
+        GeometryPrecisionReducer precisionReducer = new GeometryPrecisionReducer(precisionModel);
```

Comment:
@evansiroky this isn't blocking on a merge, but we may want to put the precisionReducer in a static field so it isn't initialized whenever this function is called.